File: weekend_exercise.py/part2/prime_number_identifier.py
number = int(input("Enter a number: "))
result = 0 


if number <= 1:
    print(f"{number} is not prime")
else:
    
    for divisor in range(2, number): 
        if (number % divisor== 0):
            result += 1 
            break 


    if (result == 0):

        print(f"{number} is prime")
    else: 

        print(f"{number} is not prime")

File: weekend_exercise.py/part2/PrimeNumberIdentifier.java
import java.util.Scanner;

public class PrimeNumberIdentifier{
public static void main(String [] args){

Scanner scanner =new Scanner (System.in);

System.out.println("Enter a number: ");
int number =scanner.nextInt();

int divisor =2;
int count =0;

while (count <=number){
    if (number % count ==0){
        count++;        
    }

} 
if (count >1){
    System.out.println(number + "is not a prime number");
}
else{
    System.out.println(number + "is not a prime number");
}







}
 } 
